spec/models/sitemap_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+
+describe Sitemap do
+  let!(:url) { 'http://www.agency.gov/boring' }
+  let!(:html) { read_fixture_file("/html/page_with_og_metadata.html") }
+  let!(:valid_attributes) { { url: url } }
+  let!(:sitemap) { Sitemap.new(valid_attributes) }
+  let!(:existing_domain) { SearchgovDomain.create!(domain: 'existing.gov') }
+
+  it { is_expected.to have_readonly_attribute(:url) }
+
+  describe 'schema' do
```

Comment:
Now that I think about it, we can move these schema specs to `Fetchable`, because these should also apply to all the classes that include `Fetchable`. The duplicate specs can be removed (if they are present) from `searchgov_url_spec.rb` and `indexed_document_spec.rb`.